data_steward/tools/create_update_drc_id_match_table.py

```diff
@@ -0,0 +1,229 @@
+    schema_list = bq.get_table_schema(IDENTITY_MATCH_TABLE)
+    ', '.join([item.name for item in schema_list])
+
+    for item in schema_list:
+        set_expression.append(SET_EXPRESSION.render(col=item.name))
+
+    # Work around to remove person_id from the fields list since person_id is required
+    # and will never need to be updated to missing_rdr
+    set_expression.remove("\nperson_id = 'missing_rdr'")
+
+    return ', '.join(set_expression)
+
+
+def update_site_drc_table(client, project_id, table_id):
+    """
+    Updates the site's DRC identity match table. If the record is empty, each field should default
+        to `missing_rdr`, else `missing_ehr` for each field that does not have any data.
+
+    :param client: bq client
+    :param project_id: the project containing the dataset
+    :param table_id: ID of the table
+    """
+    schema_list = bq.get_table_schema(IDENTITY_MATCH_TABLE)
+    fields_name_str = ',\n'.join([item.name for item in schema_list])
+
+    # person_id will not need to be updated since it will always be in table since it
+    # is a required field
+    fields = fields_name_str.replace('person_id,', '')
+    id_match_table_id = table_id
+
+    # Updates missing rows to contain 'missing_rdr'
+    missing_rdr_query = MISSING_RDR_QUERY.render(
+        project_id=project_id,
+        drc_dataset_id=DRC_OPS,
+        id_match_table_id=id_match_table_id,
+        set_expression=get_set_expression(),
+        fields=fields)
+
+    job = client.query(missing_rdr_query)
+    job.result()
+    LOGGER.info(
+        f'Updated missing row values in `{id_match_table_id}` with \'missing_rdr\''
+    )
+
+    # Updates remaining fields to contain 'missing_ehr' in site's drc id match table
+    for item in schema_list:
+        if item.name is not 'person_id':
+            missing_ehr_query = MISSING_EHR_QUERY.render(
+                project_id=project_id,
+                drc_dataset_id=DRC_OPS,
+                id_match_table_id=id_match_table_id,
+                col=item.name)
+
+            job = client.query(missing_ehr_query)
+            job.result()
+        LOGGER.info(
+            f'Updated missing column values in `{id_match_table_id}` with \'missing_ehr\''
+        )
+
+    LOGGER.info(f'Completed column update of `{id_match_table_id}` table.')
+
+
+def get_arg_parser():
+    parser = argparse.ArgumentParser(
+        description=""" Create and update DRC match table for hpo sites.""")
+    parser.add_argument('-p',
+                        '--project_id',
+                        action='store',
+                        dest='project_id',
+                        help='Project associated with the input datasets',
+                        required=True)
+    parser.add_argument('--hpo_id',
+                        action='store',
+                        dest='hpo_id',
+                        help='awardee name of the site',
+                        required=True)
+    parser.add_argument('-s',
```

Comment:
This parameter is never used. You can either remove it or add in some logging, which I think may be useful.

Reply:
You're right, I will remove that!